src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,43 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => {
+  const {
+    title,
+    user,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {comments.length > 0
```

Comment:
Prefer `Boolean(comments.length)` here

Reply:
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/110335429/a79ee64e-5375-44e4-aa2b-dc2641e2ebfe)
I did, but unfortunately linter is calling it 'redundant' (someone, teach him some manners!)).